core/controllers/classifier.py

```diff
@@ -144,6 +142,67 @@ def post(self):
 
         return self.render_json({})
 
+    @acl_decorators.open_access
+    def get(self):
+        """Handles get requests.
+
+        Retrieves trained classifier data from GCS and transfers it to frontend.
```

Comment:
Should be clearer that only the file name is being transferred to the frontend.

Would it make more sense to send all the file names for a particular exploration at once? The computation per file name looks rather intensive and it might be worth taking advantage of get_multi()-type constructs, but I am not sure.

Reply:
Done.

Regarding: sending all possible file names together, I don't think it is necessary. Also sometimes the job might not be finished when retrieving the file names initially but it might have finished by the time student arrives at a state, or it might have changed due to lazy migration, I think its better if we just stick to on-demand query. If operation of retrieving a file name is too costly then we can do it at a later point.